R/git.R

```diff
@@ -289,32 +288,22 @@ git_clean <- function() {
   paths <- st[st$status == "new", ][["file"]]
   n <- length(paths)
   if (n == 0) {
-    ui_info("Found no untracked files")
+    ui_bullets(c("i" = "Found no untracked files."))
     return(invisible())
   }
 
   paths <- sort(paths)
-  ui_paths <- map_chr(paths, ui_path)
-  if (n > 10) {
-    ui_paths <- c(ui_paths[1:10], "...")
-  }
-
-  if (n == 1) {
-    file_hint <- "There is 1 untracked file:"
-  } else {
-    file_hint <- "There are {n} untracked files:"
-  }
-  ui_line(c(
-    file_hint,
-    paste0("* ", ui_paths)
+  ui_paths <- map_chr(paths, ui_path_impl)
+  ui_bullets(c(
+    "i" = "{cli::qty(n)}There {?is/are} {n} untracked file{?s}:",
+    bulletize(usethis_map_cli(ui_paths, template = "{.file <<x>>}"))
```

Comment:
In several places I need to programmatically make bullet lists and, at the same time, apply inline styling. `usethis_map_cli()` and `bulletize()` are versions of similar functions I've written in, e.g., googledrive.